wireapp/kalium
# feat: incremental sync update user events (AR-2491)

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Incremental sync support, handling user events.
With this PR we are handling user events, related to when they change personal information, so we act on this, and update the target local user, ie: username, handle, email, avatars.

### Testing

 WIP

#### Test Coverage (Optional)


#### How to Test

You can check out this branch and use it in reloaded, you will see other users info changed automatically.
I´ll open a PR on reloaded for this updated handling

### Notes (Optional)


----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
